code/go/0chain.net/sharder/blockstore/db.go

```diff
@@ -166,79 +104,29 @@ func GetBlockWhereRecord(hash string) (*BlockWhereRecord, error) {
 	return &bwr, nil
 }
 
-//Delete metadata
-func DeleteBlockWhereRecord(hash string) {
-	bwrDB.Update(func(t *bbolt.Tx) error {
-		bkt := t.Bucket([]byte(BlockWhereBucket))
-		if bkt == nil {
-			return nil
-		}
-
-		return bkt.Delete([]byte(hash))
-	})
+// DeleteBlockWhereRecord Delete metadata.
+func DeleteBlockWhereRecord(hash string) error {
+	return bwrRedis.Delete(hash)
 }
 
-//Unmoved blocks; If cold tiering is enabled then record of unmoved blocks will be kept inside UnmovedBlockRecord bucket.
+// UnmovedBlockRecord Unmoved blocks; If cold tiering is enabled then record of unmoved blocks will be kept inside UnmovedBlockRecord bucket.
 //Some worker will query for the unmoved block and if it is within the date range then it will be moved to the cold storage.
 type UnmovedBlockRecord struct {
 	CreatedAt time.Time `json:"crAt"`
 	Hash      string    `json:"hs"`
 }
 
 func (ubr *UnmovedBlockRecord) Add() (err error) {
-	key := []byte(ubr.CreatedAt.Format(time.RFC3339))
-	value := []byte(ubr.Hash)
-
-	err = bwrDB.Update(func(t *bbolt.Tx) error {
-		bkt := t.Bucket([]byte(UnmovedBlockBucket))
-		return bkt.Put(key, value)
-	})
-
-	return
+	return bwrRedis.SetToSorted(redisSortedSetUnmovedBlock, float64(ubr.CreatedAt.UnixMicro()), ubr.Hash)
```

Comment:
`UnixMicro` is undefined in linux system. 
Generic function is required.
Should build this code with GOOS=linux.